### ISSUE TITLE
"RNLineChart" was not found in the UIManager

### ISSUE THREAD

**Issue by AndrewPulgade** [wuxudong/react-native-charts-wrapper#335]
![screen shot 2018-07-23 at 3 23 58 pm](https://user-images.githubusercontent.com/22131619/43062772-7b4ebca6-8e8c-11e8-9fd6-a0e1b1cfd392.png)

**Comment by MrZwqShuai** [comment]
I get the same problem ~ did you solve this problem？

**Comment by AndrewPulgade** [comment]
My mistake . I did not follow this documentation.

https://www.evernote.com/shard/s304/sh/5d501d94-a8e0-4309-9866-e2026356a29d/130a7aa8b84d73a6

**Comment by WaqasMumtaz** [comment]
How to solve in android plz help me ?

**Comment by edritech93** [comment]
same for me

**Comment by akinncar** [comment]
same for me

**Comment by akinnrosa** [comment]
Insert this line on top of your Podfile, and run `pod install` again
```
use_modular_headers!
```

In xcode, right click in yout project in left menu, click in "New File...", and create a empty swift file, the xcode will prompt a message 'Would you like to configure an Objective-C bridging header?' to Create Bridging Header, accept it. [(like suggests documentation) ](https://github.com/wuxudong/react-native-charts-wrapper/blob/master/installation_guide/README.md)

Don`t forget your Podfile needs Charts module:

```
  pod 'RNCharts', :path => '../node_modules/react-native-charts-wrapper'
```

This steps works for me!

**Comment by akinncar** [comment]
Insert this line on top of your Podfile, and run `pod install` again
```
use_modular_headers!
```

In xcode, right click in yout project in left menu, click in "New File...", and create a empty swift file, the xcode will prompt a message 'Would you like to configure an Objective-C bridging header?' to Create Bridging Header, accept it. [(like suggests documentation) ](https://github.com/wuxudong/react-native-charts-wrapper/blob/master/installation_guide/README.md)

Don`t forget your Podfile needs Charts module:

```
pod 'RNCharts', :path => '../node_modules/react-native-charts-wrapper'
```

This steps works for me!

**Comment by ibrahimyolbir** [comment]
React Native 0.64.2 not working for me either!

**Comment by felipegerhardt** [comment]
Not working either. Any updates?

**Comment by AMNAhhj** [comment]
any updates ?????

**Comment by Chinedu19** [comment]
Is there any solution available for expo react native?

**Comment by Arangott** [comment]
The library mentioned supporting expo. Any fix available for Expo ?

**Comment by faderik** [comment]
In my case, just rebuild the app by running `react-native run-android` fix it

**Comment by Mujahid122418** [comment]
Is there any solution available for expo react native?

**Comment by wuxudong** [comment]
Follow the tutorial in README works for me. 

```
expo init demo
cd demo
expo install react-native-charts-wrapper
eas build
```

**Comment by rzfzr** [comment]
Not sure why this and #457 were closed, I'm having the same issue "expo": "^51.0.22", I followed the readme.
